website/src/doc-demos/demo-base.js

```diff
@@ -83,22 +83,27 @@ function evalObject(source, globals, output) {
 }
 
 export function makeLayerDemo(config) {
-  const {Layer, getTooltip, props, mapStyle = MAPBOX_STYLES.LIGHT, initialViewState = INITIAL_VIEW_STATE, imports} = config;
+  const {Layer, getTooltip, props, mapStyle = true, initialViewState = INITIAL_VIEW_STATE, imports} = config;
   config.initialViewState = initialViewState;
 
-  function Demo() {
-    const _getTooltip = getTooltip && eval(getTooltip);
-    const styledGetTooltip = pickingInfo => {
-      const text = _getTooltip && _getTooltip(pickingInfo);
-      return text && {
-        text,
-        style: TOOLTIP_STYLE
-      };
+  const _getTooltip = getTooltip && eval(getTooltip);
+  const styledGetTooltip = pickingInfo => {
+    const text = _getTooltip && _getTooltip(pickingInfo);
+    return text && {
+      text,
+      style: TOOLTIP_STYLE
     };
+  };
+
+  const layerProps = evalObject(props, imports);
+
+  function Demo() {
+    const { isDarkTheme } = useColorMode();
```

Comment:
This seems to be deprecated.
<img width="504" alt="Screen Shot 2023-03-03 at 5 10 58 PM" src="https://user-images.githubusercontent.com/2461547/222865054-7c21056e-d3f3-405f-a49e-66ff5c2f5a53.png">